partner_credit_limit/migrations/11.0.0.0.2/post-migration.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    set_credit_limit_values_to_partner(cr)
+
+
+def set_credit_limit_values_to_partner(cr):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env.cr.execute(
+        """
+        SELECT id, credit_limit FROM res_partner
```

Comment:
What is the reason to use a query instead of odoo api?

Reply:
because this is a post-migration script, so after the field change values returned by the api like 
`partner.credit_limit` is False,
that why the migration script in order to recover that values from db